DevCheck.ps1

```diff
@@ -946,14 +940,14 @@ Write-Verbose("Processor...$cpu")
 $project_root = Get-ProjectRoot
 Write-Output "Windows App SDK location...$project_root"
 
-if (($CheckAll -ne $false) -Or ($CheckVisualStudio -ne $false))
-{
-    $ok = Test-VisualStudio2022Install
-    if ($ok -eq $true)
-    {
-        $null = Test-VisualStudioComponents
-    }
-}
+# if (($CheckAll -ne $false) -Or ($CheckVisualStudio -ne $false))
```

Comment:
Commented out?

Let me guess - it took a while so you commented it out?

No need.

* `devcheck -CheckDependencies` to just do the check
* `devcheck -CheckDependencies -SyncDependencies`  to check and potentially update files

That's fast to run. My testing was usually

1. `devcheck -CheckDependencies` 
2. `devcheck -CheckDependencies -SyncDependencies`
3. `devcheck -CheckDependencies` 
4. `devcheck -CheckDependencies -SyncDependencies`
5. ...

bouncing back and forth between them and seeing all work well, mixed in with actually making problems (e.g. list a package twice in a .xml) and seeing check spot it, sync fix it then check sees all is well. That sort of thing